api/envoy/type/http/v3/path_transformation.proto

```diff
@@ -20,7 +20,16 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // This API is a work in progress.
 
 message PathTransformation {
+  // Action to take on a http request when the request path url has been modified by a transformation.
+  enum NormalizePathAction {
+    DEFAULT = 0;
```

Comment:
Please add comments for each value.

Reply:
Sure, will add comments.  I am still not sure how to facilitate the change of defaults here.

Reply:
I think just point out that this is "Per-xDS client default. Envoy does ... today"

Reply:
Friendly ping

Reply:
What should be the default here? Can we make CONTINUE the default option?

Reply:
You can say something like "For Envoy clients, the DEFAULT is CONTINUE."